docs/sphinx/walkthrough/rans.rst

```diff
@@ -0,0 +1,50 @@
+.. spelling:word-list::
+
+   xy
+
+.. _rans:
+
+Wall-Modeled RANS 
+============================
+Most RANS simulations are run in steady or pseudo-steady mode. However, as AMR-Wind is a unsteady solver, 
+we try to mimic pseudo-steady mode to ensure faster convergence of the simulations. The precursor input 
+file for RANS is very similar to LES. The only modifications required are 
+
+``turbulence.model  = KLAxell``
+
+``TKE.source_terms = KransAxell``
+
+However, a LES like setup takes a lot of time to convergence and is not required. An one-dimensional vertical 
+solver is provided to generate vertical profiles for use with the RANS model (can be used with LES but not always 
+required). The solver is available at `this link <https://github.com/hgopalan/onedterrainsolver>`_.
```

Comment:
not sure I want the docs to point to personal repos. Maybe just leave out the discussion of the profile generation?

Reply:
Made it general